airbyte-integrations/connectors/source-spacex-api/source_spacex_api/spacex_api.yaml

```diff
@@ -0,0 +1,149 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+
+  requester:
+    url_base: "https://api.spacexdata.com/v4/"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "apikey"
+      api_token: "{{ config['api_key'] }}"
+
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+
+  launches_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "launches"
+      primary_key: "id"
+      path: "/launches/{{config['id'] or config['options'] or latest}}"
```

Comment:
@marcosmarxm Config['options] are now provided so that `latest`, `past`, `upcoming` options could be configured if required.